### PR TITLE
TMA-494: Support for java platform

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: gooddata.gemspec [hunk 1]
```diff
@@ -6,6 +6,9 @@ Gem::Specification.new do |s|
   s.name = 'gooddata'
   s.version = GoodData::VERSION
   s.licenses = ['BSD']
+  if RUBY_PLATFORM =~ /java/
```
Comment (yenkins) [inline gooddata.gemspec]: Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/636/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/428/ : FAILURE in 1m 47s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/736/ : SUCCESS in 1m 49s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/650/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/438/ : SUCCESS in 1m 50s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/746/ : SUCCESS in 1m 48s

**Comment by kubamahnert** [comment]
gj!

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/245/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/439/ : SUCCESS in 2m 08s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/747/ : SUCCESS in 2m 40s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1160/ : FAILURE in 3m 19s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/918/ : UNSTABLE in 5m 01s

**Comment by kubamahnert** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/250/ : SUCCESS in 22s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/448/ : SUCCESS in 2m 15s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/756/ : SUCCESS in 2m 12s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1166/ : SUCCESS in 49m 33s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/924/ : SUCCESS in 34m 13s

**Pull request merged**